### DISCUSSION TITLE
No Uninstaller

### DISCUSSION THREAD

**Discussion by Dwedit** [SimonMacer/AnWave#37] [Open Discussion]
I tested out the program to see if NVIDIA Power Management worked, it was not compatible with the system, so I wanted to uninstall the program.

Except there is no uninstaller.  It left behind registry keys and files in System32.  This is the kind of program that really needs an uninstaller.

**Comment by SimonMacer** [comment]
> I tested out the program to see if NVIDIA Power Management worked, it was not compatible with the system, so I wanted to uninstall the program.
> 
> Except there is no uninstaller. It left behind registry keys and files in System32. This is the kind of program that really needs an uninstaller.

### How to uninstall NVIDIA Power Management

AnWave > Advanced Cleanup Utility
![unia](https://github.com/user-attachments/assets/0fe52910-d810-407f-8c6b-a6feb108a7c6)

Cleanup NVIDIA Power Management (Except GPU driver) > Click [Cleanup] to uninstall NVIDIA Power Management.
![unia2](https://github.com/user-attachments/assets/e6c95576-ef4e-4a9f-ba3a-61204b8d385e)
